src/components/LoadingBar/LoadingBar.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```

Comment:
Please, rename folder and file from `LoadingBar` to `loading-bar` and use this format for other component files and folders

Reply:
Renamed